hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/migration/AccountStorageAdapterTest.java

```diff
@@ -214,7 +216,10 @@ void onDiskPropagatesInterruption() throws InterruptedException {
         willThrow(InterruptedException.class)
                 .given(virtualMapDataAccess)
                 .extractVirtualMapData(
-                        eq(onDiskAccounts), any(InterruptableConsumer.class), eq(32));
+                        getStaticThreadManager(),
```

Comment:
I'm surprised this works. Doesn't mockito usually want an `eq()` around `getStaticThreadManager()` if there are `eq()` or `any()` on other params?

Reply:
It did not work. 🙂  There were few other test failures. Fixed some of them